pkg/operator/ceph/agent/agent.go

```diff
@@ -201,30 +201,6 @@ func (a *Agent) discoverFlexvolumeDir() (flexvolumeDirPath, source string) {
 		return getDefaultFlexvolumeDir()
 	}
 
-	// unmarshal to a NodeConfigKubelet
-	configKubelet := NodeConfigKubelet{}
```

Comment:
do these types not even exist anymore in the new 1.11 client-go?

Reply:
no, i couldn't find them. I'll take another look to be sure.

Reply:
I don't see the `VolumePluginDir` anywhere, and there was also discussion in a [previous issue](https://github.com/rook/rook/issues/1282#issuecomment-348640472) about this property not being included anymore. So this code must not have been exercised anyway since it hasn't been possible to configure. We really just rely on setting the `FLEXVOLUME_DIR_PATH` env var in the operator yaml as described [here](https://rook.io/docs/rook/v0.8/flexvolume.html#configuring-the-rook-operator. I'll clean up this function a bit better to make that more clear.